AppKit/CPStringDrawing.j

```diff
@@ -59,22 +57,12 @@ CPStringSizeCachingEnabled = YES;
     return [self sizeWithFont:aFont inWidth:NULL];
 }
 
-- (void) _initializeStringSizing
++ (void) initialize
```

Comment:
-  +initialize should not  be in a category but in the main class implementation (for clarity).
-  You don't need to call super because +initialize is already sent to the superclass.
-  If you don't need +initialize to be sent to super, you need to block it in the subclass, otherwise it will be sent to both the subclass and  the superclass with the subclass impl (this one). So wrap this into a check  `if ([self class] == [CPString class) { }`